configure

```diff
@@ -728,43 +730,46 @@ if [ $cbytes -gt 0 ]; then
     ocamlfind_archives="bytes.cma ${ocamlfind_archives}"
 fi
 
-echo "# Makefile.config written by configure" >Makefile.config
-echo "OCAML_CORE_STDLIB=${ocaml_core_stdlib}" >>Makefile.config
-echo "OCAML_CORE_BIN=${ocaml_core_bin}" >>Makefile.config
-echo "OCAML_CORE_MAN=${ocaml_core_man}" >>Makefile.config
-echo "OCAML_SITELIB=${ocaml_sitelib}" >>Makefile.config
-echo "FINDLIB_PATH=${ocamlpath}" >>Makefile.config
-echo "OCAML_THREADS=${ocaml_threads}" >>Makefile.config
-echo "OCAMLFIND_BIN=${ocamlfind_bin}" >>Makefile.config
-echo "OCAMLFIND_MAN=${ocamlfind_man}" >>Makefile.config
-echo "OCAMLFIND_CONF=${ocamlfind_config}" >>Makefile.config
-echo "OCAMLFIND_OCAMLFLAGS=${ocamlfind_ocamlflags}" >>Makefile.config
-echo "OCAMLFIND_ARCHIVES=${ocamlfind_archives}" >>Makefile.config
-echo "OCAML_AUTOLINK=${ocaml_autolink}" >>Makefile.config
-echo "OCAML_REMOVE_DIRECTORY=${have_remdir}" >>Makefile.config
-echo "EXEC_SUFFIX=${exec_suffix}" >>Makefile.config
-echo "LIB_SUFFIX=${lib_suffix}" >>Makefile.config
-echo "CUSTOM=${custom}" >>Makefile.config
-echo "PARTS=${parts}" >>Makefile.config
-echo "INSTALL_TOPFIND=${with_topfind}" >>Makefile.config
-echo "USE_CYGPATH=${use_cygpath}" >>Makefile.config
-echo "HAVE_NATDYNLINK=${have_natdynlink}" >>Makefile.config
-echo "VERSION=${version}" >>Makefile.config
-echo "ENABLE_TOPFIND_PPXOPT=${enable_topfind_ppxopt}" >>Makefile.config
-echo "SYSTEM=${system}" >>Makefile.config
-echo "NUMTOP=${numtop}" >>Makefile.config
-echo "SH=${sh}" >>Makefile.config
-if [ "$mingw_lib" != "" ]; then
-    echo "OCAMLC_FLAGS=-I \"${mingw_lib}\"" >>Makefile.config
-    echo "OCAMLOPT_FLAGS=-I \"${mingw_lib}\"" >>Makefile.config
-fi
-echo "OPAQUE=${opaque}" >>Makefile.config
-echo "OCAMLOPT_G=${native_debugging_info}" >>Makefile.config
-echo "CHECK_BEFORE_INSTALL=${check_before_install}" >>Makefile.config
-echo "INSTALLDIR = install -d" >>Makefile.config
-echo "# change to INSTALLDIR = mkdir -p   when BSD install is unavavailable" >>Makefile.config
-echo "INSTALLFILE = install -c" >>Makefile.config
-echo "# change to INSTALLFILE = cp   when BSD install is unavailable" >>Makefile.config
+{
+    echo "# Makefile.config written by configure" >Makefile.config
```

Comment:
double redirection

Reply:
I had this to make sure not to append to an existing `Makefile.config` but as you point out, redirecting the whole thing via `>` makes more sense. Somehow missed the forest for the trees.